README.md

```diff
@@ -151,11 +128,11 @@ provide test coverage reports.
 To check the test coverage: 
 
 ```shell script 
-./go tests:coverage
+npm run tests:coverage
 ```
 
-`solidity-coverage` runs its own instance of Ganache internally, as well as
-instrumenting contracts before running.
+`solidity-coverage` runs its own instance of Ganache internally, as well as instrumenting contracts before running.
```

Comment:
Remove reference to Ganache. This is only applicable when using truffle. According to the docs (https://github.com/sc-forks/solidity-coverage), it runs on directly on the HardhatEVM when hardhat is used.

Reply:
updated.